fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/call/CallFunc.java

```diff
@@ -28,13 +29,14 @@ public abstract class CallFunc {
     /**
      * Get the instance of CallFunc
      */
-    public static CallFunc getFunc(UserIdentity user, UnboundFunction unboundFunction) {
+    public static CallFunc getFunc(ConnectContext ctx, UserIdentity user, UnboundFunction unboundFunction) {
         String funcName = unboundFunction.getName().toUpperCase();
         switch (funcName) {
-            case "EXECUTE_STMT":
+            // TODO, built-in functions require a separate management
```

Comment:
always forward call procedure command to master because there maybe insert and select sql in one store procedure. Doris could not ensure the select sql could read the data inserted before. There maybe data consistency problem.

Reply:
call stmt already forward to master
```CallCommand extends Command implements ForwardWithSync {```